packages/language-server/src/completion-items.ts

```diff
@@ -199,8 +201,18 @@ function createTextEdits(
       // the attribute value syntax exists
       /* istanbul ignore next */
       range = getXMLAttributeValueRange(suggestion.astNode) ?? range;
-      newText = `"${suggestion.ui5Node.name}"`;
       // Attribute values should contain quotation marks
+      newText = `"${newText}"`;
+      filterText = newText;
+      break;
+    }
+    case "BooleanValueInXMLAttributeValueCompletion": {
```

Comment:
what would be the difference between suggesting boolean/number/string ? why is a special case needed?

Reply:
If its the quotes they can be part of the literal suggestions could they not be?

Reply:
actually they probably should be part of the literal suggestion as if I read this correctly we will always offer double quotes here even if the user started entering single quotes?

Reply:
Are single quotes valid in xml attributes?
We put double quotes on all suggested attribute values (including enum member and namespaces), replacing the original quotes... I did this because the token of the value includes the quotes, but if we add a token for the inner string we can change this.

Reply:
> what would be the difference between suggesting boolean/number/string ? why is a special case needed?

No real difference (maybe except escaping string values when we have them), but it's like we have different cases when suggesting properties, events and aggregations.
Technically I think the icons might be different. Boolean values are keywords or constants (I used constants because I don't think they are considered keywords in xml), while strings and numbers are values.